packages/keyring-api/tsconfig.build.json

```diff
@@ -3,11 +3,9 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "dist",
-    "rootDir": "src",
-    // FIXME: We should investigate how to get rid of this flag, as this will turn off type
```

Comment:
This is no longer required since the `snaps-*` deps have been moved to the new `keyring-snap-*` packages.